uxarray/__init__.py

```diff
@@ -1,2 +1,17 @@
 from .grid import *
 from .helpers import *
+
+
+def __version__():
+    # Attempt to import the needed modules
+    try:
+        from importlib.metadata import version as version
+    except ImportError:
+        from importlib_metadata import version as version
+
+    try:
+        __version = version("uxarray")
+        print(__version)
```

Comment:
We want to avoid directly printing the version. We instead would like to return a string of the current version.